pkg/controller/logandevent/logandevent.go

```diff
@@ -0,0 +1,42 @@
+package logandevent
```

Comment:
Probably we can skip `and` in the middle for better read and call package as `eventlog` or anything else

Reply:
fine with `eventlog`